package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@elastic/package-registry",
+  "version": "0.2.0",
+  "description": "Elastic Package Registry (EPR)",
+  "repository": "git@github.com:elastic/package-registry.git",
+  "author": " Nicolas Ruflin <spam@ruflin.com>",
+  "license": "Elastic-License",
```

Comment:
`npm`/`yarn` print a warning because this is not available in the [SPDX License List](https://spdx.org/licenses/). Perhaps we could create a PR to be included. [Mongo's SSPL is in the list](https://spdx.org/licenses/SSPL-1.0.html)

Reply:
Or we make the definition itself Apache licensed?

Reply:
Maybe when/if we split types out to their own project/repo. Or we should change the name/files. This is called `@elastic/package-registry` and actually brings in all the other repo files so the Elastic license is appropriate for what it's currently doing.

Reply:
👍